### ISSUE TITLE
Many keys are broken on linux, making the editor practically unusable

### ISSUE THREAD

**Issue by destenson** [Microsoft/vscode#13332]
- VSCode Version:
Version 1.5.3
Commit 5be4091987a98e3870d89d630eb87be6d9bafd27
Date 2016-09-22T08:40:08.699Z
Shell 0.37.6
Renderer 49.0.2623.75
Node 5.10.0

- OS Version:
Ubuntu 16.04.1

Steps to Reproduce:

1. Open a file in the editor.
2. Move the cursor away from the beginning of the file.
3. Attempt to use the left arrow key to move the cursor left or attempt to use the DEL button to delete something, or page up, page down, backspace, spacebar, etc.

This egregious lack of keyboard functionality must be some issue with my environment or something. How in the world can I troubleshoot this to fix it? All of the keys work normally in every other program on my system. They even work properly in Code inside the Developer Tools window, and in other parts of the program like Explorer, but not in the editor.

**Comment by Tyriar** [comment]
@destenson can you try the insiders build to see if it reproduces there? https://code.visualstudio.com/insiders

**Comment by destenson** [comment]
After noticing a lot of errors and warnings in the developer tools window, I decided to get rid of them. It turns out I had installed a lot of the available extensions & several of them conflicted & probably some handled, or mishandled, the key press events. Once I removed a bunch of them, the problem went away. This is not a bug per se, but it does show a need to enhance vscode with better error-detection and handling of conflicts between extensions.
